src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -15,10 +15,15 @@ import "./SpacesDropdown.scss";
 
 const noneOptionName = "None";
 const spaceNoneOption = Object.freeze({
-  none: Object.freeze({ id: undefined, name: "", template: undefined }),
+  none: Object.freeze({ id: "", name: "", template: undefined }),
```

Comment:
I've set an empty string here to get rid of the warning. React requires all the list items to have a key prop defined.